dev/a11y_assessments/lib/utils.dart

```diff
@@ -0,0 +1,7 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/services.dart';
+
+Color appSeedColor = const Color(0xff6750a4);
```

Comment:
```suggestion
const Color appSeedColor = Color(0xff6750a4);
```

Reply:
also why a hardcode value vs using something like Theme.of(context).primaryColor

Reply:
Updated to use Theme.of(context).primary.

Reply:
in that case is this still needed?

Reply:
Removed appSeedColor